pom.xml

```diff
@@ -229,20 +229,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
I believe Javadoc is required in order to publicly release.

Reply:
yes, javadoc is still there. this was a duplicate entry